middlewares/validators/user.js

```diff
@@ -66,9 +68,18 @@ const validateJoinData = async (req, res, next) => {
       country: joi.string().min(1).required(),
       foundFrom: joi.string().min(1).required(),
       introduction: joi.string().min(1).required(),
```

Comment:
On introduction also validate based on word length

Reply:
For how many words?

Reply:
100 words